plugin/src/main/groovy/com/btkelly/gnag/utils/ProjectHelper.groovy

```diff
@@ -29,7 +30,7 @@ class ProjectHelper {
         this.project = project
     }
 
-    Project getProject() {
+    DefaultProject getProject() {
```

Comment:
TIL.

Reply:
I don't know if I actually needed this come to think, I was getting an `AbstractMethodError` when attempting to use this class and I thought this would fix it. I was able to fix it but I'm not sure if this was the actual fix 🤔